kinto/core/testing.py

```diff
@@ -34,9 +34,9 @@ def __init__(self, *args, **kwargs):
         self.authn_type = 'basicauth'
         self.prefixed_userid = 'basicauth:bob'
         self.effective_principals = [
-            self.prefixed_userid,
             'system.Everyone',
-            'system.Authenticated']
+            'system.Authenticated',
+            'bob']
```

Comment:
Any idea why you want to change the behavior here? Is there a security risk not to have the prefix in the principal here.